packages/manager/apps/dedicated/client/app/account/user/components/newAccountForm/new-account-form-controller.js

```diff
@@ -300,17 +294,29 @@ export default class NewAccountFormController {
     let promise = this.userAccountServiceInfos
       .updateUseraccountInfos(model)
       .then((result) => {
+        const tracking = {
+          name: `dedicated::account::user::infos_${
+            result !== 'null' ? 'error' : 'success'
+          }`,
+          type: 'navigation',
+        };
+        if (!this.coreConfig.isRegion('US')) {
+          const smsConsent =
+            typeof this.model.smsConsent !== 'undefined'
```

Comment:
Is it normal that `null` or `undefined` values are strings ?

Reply:
I think, it was handled this way before